enterprise/server/remote_execution/copy_on_write/copy_on_write.go

```diff
@@ -879,11 +879,12 @@ func ConvertFileToCOW(ctx context.Context, env environment.Env, filePath string,
 	}
 
 	var chunksMu sync.Mutex
+out:
 	for chunkStartOffset := int64(0); chunkStartOffset < totalSizeBytes; chunkStartOffset += chunkSizeBytes {
 		select {
 		case <-egCtx.Done():
 			// One goroutine failed - exit the for loop
-			break
+			break out
```

Comment:
Do you mind either factoring this out to a separate function or adding a control variable and moving the `default` logic out of the `select` statement so we don't need to use the labelled loop here?